src/Makefile

```diff
@@ -133,6 +133,16 @@ jenkins-parallel: clean
 	$(MAKE) check-style
 	$(MAKE) test-vdo-distribution
 
+.PHONY: dmlinux-jenkins
+dmlinux-jenkins:
+	$(MAKE) -j$(NUM_JOBS) dmlinux-jenkins-parallel
+dmlinux-jenkins-parallel: export VDO_TESTINDEX=$(TEST_INDEX)
+dmlinux-jenkins-parallel: clean
+	$(MAKE) all
+	$(MAKE) -C packaging/dmlinux clean all
+	@echo "Using index name of $$VDO_TESTINDEX"
```

Comment:
You sohuldn't need "clean" here; the clean target should have done that for you. You also might not need "all" (since it should be the default, or you can set it to be).

Reply:
See above on my thoughts

Reply:
If dmlinux realy is going to be a separate thing, you shouldn't need the dependence on the "clean" target, and you shouldn't nee  the generic "make all" before building the dmlinux stuff. Maybe it'll be clearer once you update it.

Reply:
I was trying to say its not part of the normal build and test of vdo-devel. the dmlinux target itself still needs all the other things to be built (I think) so that the testing will work after we build the dmlinux source rpm. So I'm saying I don't think the dmlinux packaging dir belongs in the packaging Makefile (the default build and test) but instead should just be handled in the particular dmlinux target.

Reply:
Okay, I guess this seems fine.